c2pa_c_ffi/src/c_api.rs

```diff
@@ -603,15 +603,16 @@ pub unsafe extern "C" fn c2pa_reader_json(reader_ptr: *mut C2paReader) -> *mut c
 ///
 /// # Safety
 /// reader_ptr must be a valid pointer to a C2paReader.
+/// The returned value MUST be released by calling c2pa_string_free.
 #[no_mangle]
-pub unsafe extern "C" fn c2pa_reader_remote_url(reader_ptr: *mut C2paReader) -> *const c_char {
+pub unsafe extern "C" fn c2pa_reader_remote_url(reader_ptr: *mut C2paReader) -> *mut c_char {
```

Comment:
This should be treated as a constant by the caller, making any modification would be a very bad thing. Why is the caller treating it as mutable?